.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: Running tests
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ios_test:
+    name: Running tests in iOS
+    runs-on: macOS-12
+    defaults:
+      run:
+        working-directory: ./ios
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup Java JDK
+      uses: actions/setup-java@v2.1.0
+      with:
+       distribution: 'adopt'
+       java-version: '11'
+
+    - uses: actions/cache@v2
+      id: bunlderCache
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+
+    - name: Bundle install
+      run: bundle install --path vendor/bundle
+
+    - name: Cache Pods
+      uses: actions/cache@v2
+      id: cocoapodCache
+      with:
+        path: Pods
+        key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-pods-
+
+    - name: Generate KMM frameworks for Cocoapods
+      run: |
+        cd ..
+        ./gradlew generateDummyFramework
+
+    - name: Install Pods Dependencies
+      run: bundle exec pod install
+
+    - name: Build and Test
+      run: bundle exec fastlane buildAndTest
```

Comment:
Will we post the code coverage result on the PR

Reply:
@phongvhd93 I will do it in this chore #53 🙏